scripts/amd/gemm/tune_gemm.py

```diff
@@ -22,12 +23,12 @@ def get_full_tuning_space():
     block_k_range = [16, 32, 64, 128, 256]
     split_k_range = [1, 2, 4, 5, 6, 8, 10, 12, 16, 18, 24]
     num_warps_range = [1, 2, 4, 8]
-    group_m_range = [1, 4, 8]
+    group_m_range = [1, 4, 8, 16, 32]
     # For now we see better perf with num_stages=0 for all gemm configs we care
     # But keep this explicit so that we do not forget we may need to set it to
     # other values in the future
-    num_stage_range = [1, 0]
-    waves_per_eu_range = [0,1,2,3,4]
+    num_stage_range = [0]
+    waves_per_eu_range = [0]
```

Comment:
Why is this reduced from [0,1,2,3,4] to [0]?

Reply:
Because 
1. It saves tuning time
2. For gemm, the occupancy is always large. We need manually investigation for the best value for waves-per-eu if needed